src/apps/frontend/pages/login/otp.page.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useEffect, useState } from 'react';
+import { Block } from 'baseui/block';
+import { useFormik } from 'formik';
+import { OTPSchema } from '../../schemas/phone-signup-schema';
+import { FormControl } from 'baseui/form-control';
+import { Input } from 'baseui/input';
+import {
+  HeadingMedium,
+  HeadingSmall,
+  ParagraphXSmall,
+} from 'baseui/typography';
+import { Button } from 'baseui/button';
+import { useNavigate } from 'react-router-dom';
+
+const initialValues = {
+  otp: '',
+};
+
+const OtpVerification = () => {
+  const [loginStatus, setloginStatus] = useState(false);
+
+  const checkStatus = () => {
+    if (loginStatus) {
+      navigate('/');
+    }
+  };
+
+  useEffect(() => {
+    checkStatus();
+  }, [loginStatus]);
+
+  const navigate = useNavigate();
+
+  const formik = useFormik({
+    initialValues: initialValues,
+    validationSchema: OTPSchema,
+    onSubmit: (values, action) => {
+      console.log(values);
+      action.resetForm();
+      setloginStatus(true);
+    },
+  });
+
+  return (
+    <Block
+      overrides={{
+        Block: {
+          style: ({ $theme }) => ({
+            display: 'flex',
+            backgroundColor: $theme.colors.accent100,
+            width: '100vw',
+            height: '100vh',
+          }),
+        },
+      }}
+    >
```

Comment:
You can directly apply some styles to Block without using overrides for cleaner code.
<Block width={''} ></Block>